di_website/templates/includes/cards/card-blog.html

```diff
@@ -10,7 +10,9 @@
 
     <div class="card__body">
       {% if blog.published_date %}
-        <span class="card__meta">{{ blog.published_date|date:"j F Y" }}</span>
+        {% if content_type != 'resources' %}
```

Comment:
Use a more specific variable that doesn't result in confusing usages ... i.e `hide_date` ... then check `if not hide_date`

Besides the actual card templates, the only other change should be in `related_content.html` where you'll set `hide_date` to `True`

Reply:
@edwinmp added `hide_date` to publication models and templates